website/layouts/partials/hooks/body-end.html

```diff
@@ -3,3 +3,25 @@
 <script src="{{ $jsCopy.RelPermalink }}"></script>
 {{ end }}
 
+<script src="https://cdn.jsdelivr.net/npm/@docsearch/js@3"></script>
+
+<script type="text/javascript">
+
+docsearch({
+
+appId: 'A1FXAERJE4',
+
+apiKey: '88f3a753878f24ffdcd3d786be775545',
```

Comment:
is this apikey vulnerable to being misused by other people if it is being exposed here? Does angolia have some reccomendation for the visibility of this api-key?

Reply:
no. This is a public API key that is meant to be used in front end code, It can be regenerated at any time. Used for search queries and to list the indices we have access to.